core/templates/pages/about-page/about-page.module.ts

```diff
@@ -69,7 +69,7 @@ const bootstrapFn = (extraProviders: StaticProvider[]) => {
 };
 const downgradedModule = downgradeModule(bootstrapFn);
 
-declare var angular: any;
+declare var angular: ng.IAngularStatic;
```

Comment:
Could you please explain what are we doing here like what is ng.IAngularStatic? Also, shouldn't this be extended to all page.modules?

Reply:
First I checked the type of angular in controllers and directives, it was `ng.IAngularStatic`. I assigned the same type to angular in this module and it worked executed without any errors. So, I guess this is the correct type. Yup, this needs to be done in all the modules.

Reply:
Okay makes sense. So, lets do it for all files then?

Reply:
Done!